<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>udg::QCreateDicomdirBase</class>
 <widget class="QDialog" name="udg::QCreateDicomdirBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create DicomDir</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string/>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1" >
            <widget class="QLabel" name="m_labelMbCdDvdOcupat" >
             <property name="autoFillBackground" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>650 Mb</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QProgressBar" name="m_progressBarOcupat" >
             <property name="maximum" >
              <number>100</number>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
             <property name="textVisible" >
              <bool>false</bool>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="textDirection" >
              <enum>QProgressBar::TopToBottom</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2" >
            <widget class="QLabel" name="m_labelSizeOfDicomdir" >
             <property name="font" >
              <font>
               <family>Sans Serif</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text" >
              <string>Mida Dicomdir :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2" >
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="m_lineEditDicomdirPath" />
             </item>
             <item>
              <widget class="QPushButton" name="m_buttonExamineDisk" >
               <property name="text" >
                <string/>
               </property>
               <property name="icon" >
                <iconset resource="../main/main.qrc" >:/images/folderclose.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2" >
            <widget class="QLabel" name="m_labelCreateDicomdir" >
             <property name="font" >
              <font>
               <family>Sans Serif</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text" >
              <string>Create Dicomdir at :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2" >
            <widget class="QComboBox" name="m_comboBoxAction" >
             <item>
              <property name="text" >
               <string>Copy to disk or external device</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Burn in a Cd</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Burn in a Dvd</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_buttonCreateDicomdir" >
            <property name="toolTip" >
             <string>Creates the Dicomdir with the selected studies</string>
            </property>
            <property name="text" >
             <string>Create Dicomdir</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>381</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="GroupBox1" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="m_dicomdirStudiesList" >
        <property name="minimumSize" >
         <size>
          <width>403</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Study</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Patient ID</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Patient Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Modality</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Study Date</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Study Time</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>studyUID</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_buttonRemove" >
          <property name="toolTip" >
           <string>Remove selected study of the list</string>
          </property>
          <property name="text" >
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>31</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_buttonRemoveAll" >
          <property name="toolTip" >
           <string>Removel all the studies of the list</string>
          </property>
          <property name="text" >
           <string>Remove All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections/>
</ui>
